### ISSUE TITLE
CI should fail when a file is missing

### ISSUE THREAD

**Issue by quaertym** [ember-cli/ember-cli#5385]
I forgot to add a component to git however travis CI passed. This is Travis build log. This should break the build.
```
ok 219 PhantomJS 1.9 - Global error: Error: Assertion Failed: A helper named 'editable-input' could not be found at http://localhost:7357/assets/vendor.js, line 16067
    ---
        Log: |
            { type: 'error',
              text: 'Error: Assertion Failed: A helper named \'editable-input\' could not be found at http://localhost:7357/assets/vendor.js, line 16067\n' }
```

**Comment by rwjblue** [comment]
Can you provide a repo that reproduces (to make tracking down the issue easier)?

**Comment by rwjblue** [comment]
Can you provide a repo that reproduces (to make tracking down the issue easier)?

**Comment by rwjblue** [comment]
The issue here is that global errors are reported per test and end up being recorded as "passing" (WAT). They should be failing though, which would cause our run to result in a non-zero exit code.

@johanneswuerbach - Thoughts?

**Comment by stefanpenner** [comment]
@quaertym can you share which versions of stuff you are using?

**Comment by rwjblue** [comment]
@stefanpenner - I'm working on a small repro, but it happens on master

**Comment by rwjblue** [comment]
Demo here: https://github.com/rwjblue/global-failures-not-failing-tests#global-failures-not-failing-tests.

Basically, `app.import` something that `throw new Error('Boom!');`, then run `ember test`.  Exit code is zero.

**Comment by quaertym** [comment]
@rwjblue Thank you. I tried but could not reproduce this in a fresh repo. 
@stefanpenner Versions: ember-cli `1.13.15`, this is .travis.yml:
```
---
language: node_js
node_js:
  - "0.12"

sudo: false

cache:
  directories:
    - node_modules
    - bower_components

before_install:
  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
  - "npm config set spin false"
  - "npm install -g npm@^2"

install:
  - npm install -g bower
  - npm install
  - bower install

script:
  - npm test

```

**Comment by johanneswuerbach** [comment]
Thanks for the report, should be fixed in testem `v1.1.0`. The new version can be fetched by `rm -rf node_modules && npm install`

**Comment by quaertym** [comment]
@johanneswuerbach Thanks, I have tried with @rwjblue 's repo and it works. :+1:
